packages/embark-ui/src/components/ContractOverview.js

```diff
@@ -171,12 +186,27 @@ class ContractFunction extends Component {
                : (ContractFunction.isPureCall(this.props.method) &&
                    this.makeBadge('success', 'white', 'call')) ||
                    this.makeBadge('warning', 'black', 'send')}
+              {ContractFunction.isPayable(this.props.method) &&
+                this.makeBadge('light', 'black', 'payable')
+              }
             </div>
           </CardTitle>
         </CardHeader>
         {!ContractFunction.isFallback(this.props.method) &&
          <Collapse isOpen={this.state.functionCollapse} className="relative">
           <CardBody>
+            {ContractFunction.isPayable(this.props.method) &&
+              <Form inline>
+                <Label for={this.props.method.name + '-value'} className="mr-2 font-weight-bold contract-function-input">
+                  Ether
```

Comment:
I'd rename this `Value in ETH`

Reply:
Since there's multiple units (Wei, GWei, Gether, Tether, etc..) I thought Ether might be best, but can switch to `ETH`. If we do introduce a dropdown for other units in the future, we should go back to `Ether` though.